source/tutorials/fmt-cdiff.rst

```diff
@@ -0,0 +1,478 @@
+
+Finally, ordination is a popular approach for exploring microbial community composition in the context of sample metadata. We can use the `Emperor`_ tool to explore principal coordinates (PCoA) plots in the context of sample metadata. While our ``core-metrics-phylogenetic`` command did already generate some Emperor plots, we want to pass an optional parameter, ``--p-custom-axes``, which is very useful for exploring time series data. The PCoA results that were used in ``core-metrics-phylogeny`` are also available, making it easy to generate new visualizations with Emperor. We will generate Emperor plots for unweighted UniFrac and Bray-Curtis so that the resulting plot will contain axes for principal coordinate 1, principal coordinate 2, and days since the experiment start. We will use that last axis to explore how these samples changed over time.
+
+.. command-block::
+
+   qiime emperor plot \
+     --i-pcoa core-metrics-results/unweighted_unifrac_pcoa_results.qza \
+     --m-metadata-file sample-metadata.tsv \
+     --p-custom-axes animations_gradient \
+     --o-visualization core-metrics-results/unweighted-unifrac-emperor-animations_gradient.qzv
+
+   qiime emperor plot \
+     --i-pcoa core-metrics-results/bray_curtis_pcoa_results.qza \
+     --m-metadata-file sample-metadata.tsv \
+     --p-custom-axes animations_gradient \
+     --o-visualization core-metrics-results/bray-curtis-emperor-animations_gradient.qzv
+
+.. question::
+    Do the Emperor plots support the other beta diversity analyses we've performed here? (Hint: Experiment with coloring points by different metadata.)
+
+.. question::
+    What differences do you observe between the unweighted UniFrac and Bray-Curtis PCoA plots?
+
+Alpha rarefaction plotting
+--------------------------
+
+In this section we'll explore alpha diversity as a function of sampling depth using the ``qiime diversity alpha-rarefaction`` visualizer. This visualizer computes one or more alpha diversity metrics at multiple sampling depths, in steps between 1 (optionally controlled with ``--p-min-depth``) and the value provided as ``--p-max-depth``. At each sampling depth step, 10 rarefied tables will be generated, and the diversity metrics will be computed for all samples in the tables. The number of iterations (rarefied tables computed at each sampling depth) can be controlled with ``--p-iterations``. Average diversity values will be plotted for each sample at each even sampling depth, and samples can be grouped based on metadata in the resulting visualization if sample metadata is provided with the ``--m-metadata-file`` parameter.
+
+.. command-block::
+
+   qiime diversity alpha-rarefaction \
+     --i-table table.qza \
+     --i-phylogeny rooted-tree.qza \
+     --p-max-depth 4000 \
```

Comment:
Why only to 4000? Maybe it makes sense to go all the way for this tutorial, that way users can see everything.